cert/spec.go

```diff
@@ -291,19 +262,15 @@ func (spec *Spec) HasChangedOnDisk() (bool, bool, error) {
 	return false, false, nil
 }
 
-// CheckDiskPKI checks the PKI information on disk against cert spec and alerts upon differences
+// checkDiskPKI checks the PKI information on disk against cert spec and alerts upon differences
 // Specifically, it checks that private key on disk matches spec algorithm & keysize,
 // and certificate on disk matches CSR spec info
-func (spec *Spec) CheckDiskPKI() error {
-	certPath := spec.Cert.Path
-	keyPath := spec.Key.Path
+func (spec *Spec) checkDiskPKI(cert *x509.Certificate, keyData []byte) error {
+	// first, just verify that the cert presented is actually still validly signed by the CA.
```

Comment:
Is there a missing check here? Looks like the following logic just checks algorithm and keysize, not if cert if validly signed by CA. Or maybe remove this comment if it's not relevant anymore?

Reply:
That comment shouldn't be there; the actual CA/cert validation is handled by https://github.com/cloudflare/certmgr/blob/79caef83c1d0dfcfd6c585a2911261c0f59605d4/cert/spec.go#L385 , but this comment is left over from me moving things around trying to remove the KP persistence reliance.

I'll fix shortly.